src/logic/safe/store/selectors/index.ts

```diff
@@ -147,12 +147,15 @@ export const safeActiveAssetsSelector = createSelector(
   },
 )
 
-export const safeActiveAssetsListSelector = createSelector(safeActiveAssetsSelector, (safeList) => {
-  if (!safeList) {
-    return Set([])
-  }
-  return Set(safeList)
-})
+export const safeActiveAssetsListSelector = createSelector(
+  safeActiveAssetsSelector,
+  (safeList): Set<string> => {
```

Comment:
is there a type for SafeList?

Reply:
actually this is not a safe list, it's an asset list